portal/app/farm_updater/std_api.py

```diff
@@ -26,9 +26,32 @@ def get_devices_status(self):
         resp = requests.get(self.url + self.DEVICE_ENDPOINT, cookies=self.cookie)
         return resp.json()
 
+    def get_single_device_status(self, dev_id):
+        try:
+            resp = requests.get(self.url + self.DEVICE_ENDPOINT + dev_id, cookies=self.cookie)
+        except Exception as exc:
```

Comment:
what exception is `requests.get` going to raise? If any, catch a more specific exception, like ValueError or TypeError, or whatever exception it could be.